editions/tw5.com/tiddlers/widgets/SetWidget.tid

```diff
@@ -120,3 +120,25 @@ src='<$set name="myTiddler" value="HelloThere">
 </$set>'/>
 
 <<<
+
+!! Using the Set Widget to Create Global Variables
+
+There are times when it makes sense to use the features of the [[SetWidget]] rather than procedures or functions to create global variables.  This can be accomplished by placing the set variable widget in a tiddler that is tagged [[$:/tags/Global|SystemTag: $:/tags/Global]].  If multiple variables are required, the set variable widget can be nested as shown here:
+
+<<<
+<div class="doc-example">
+
+```
+<$set name="myglobalvariable" value="I am global">
+
+    <$set name="myotherglobalvariable" value="I am also a global variable.">
+
+    </$set>
+
+</$set>
+
+```
+</div>
+<<<
+
+The line spacing indicated is necessary for the set variable widget to be parsed correctly.
```

Comment:
@Jermolene this sounds like a regression. I recall that we changed to parsing inline in $importvariables to avoid such issues.

Reply:
OK, I think it isn't quite a regression but we should be able to improve the brittleness due to whitespace by setting the parser to trim whitespace in $importvariables.

Reply:
Thanks @saqimtiaz I was confused by that, I expected that 75d10a2dc3f240755c43d17f225d39f482b8f1b4 would have fixed this. But of course the problem is not extraneous paragraphs, it's that extraneous whitespace appears in the output.

So I think it's probably best to recommend using `\whitespace trim`. For example:


 ```
\whitespace trim
 <$set name="myglobalvariable" value="I am global">
     <$set name="myotherglobalvariable" value="I am also a global variable.">
     </$set>
 </$set>
 ```

Reply:
Changed example text to conform. Added note pointing out use of pragma whitespace trim.